Source/GSXib5KeyedUnarchiver.m

```diff
@@ -2192,6 +2201,11 @@ - (id) decodeCellContentsForElement: (GSXibElement*)element
     {
       object = [element attributeForKey: @"stringValue"];
     }
+  else if ([class isSubclassOfClass: [NSPathCell class]])
+    {
+      GSXibElement *el = [element elementForKey: @"url"];
```

Comment:
As URLs may appear in other context as well we should add a general way to decode it. I think the way to go here is the dictionary XmlTagToObjectClassMap, where we could define a mapping from "url" to NSURL. But I am unsure whether this shouldn't work automatically?

Reply:
This does work.   Should we leave it or come up with something better?

Reply:
I really would prefer to change it. The idea of this class is that we decode each element for itself and construct bigger entities from these components. Here we are reaching through. Sometimes there is no way around that, but here there is.

Reply:
Handling this in a separate PR.

Reply:
Does this just mean you want to merge the code as is and clean up later?